packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -73,10 +78,11 @@ const CardSection = ({ title, cards, theme }) => {
                     <Card
                       key={index}
                       image={card.image}
-                      title={card.title}
-                      content={card.copy}
-                      href={card.link.href}
-                      target={card.link.target}
+                      title={card.heading}
+                      eyebrow={card.eyebrow}
+                      copy={card.copy}
+                      href={card.cta.href}
+                      target={card.cta.target}
```

Comment:
@mkothur see @photodow's comment here
https://github.com/carbon-design-system/ibm-dotcom-library/pull/1430/files#r381782379

Reply:
yeah @kennylam now this branch is rebased with latest.